src/server.js

```diff
@@ -29,13 +28,11 @@ function sniffMethod(label, fn) {
             });
 
             if (callback) {
-                return callback.apply(self, arguments);
+                return callback.apply(this, arguments);
             }
         });
 
-        req.on('error', (err) => {
-            console.error(err);
-        });
+        req.on('error', console.log);
```

Comment:
Jaki log? Error ma być :)

Reply:
fixed in https://github.com/piecioshka/xhr-sniffer/pull/2/commits/b6eba68112e151b648b2f7315a21e6870fff6ebd